packages/connect-agreement/src/types.ts

```diff
@@ -19,13 +21,30 @@ export interface VersionData {
   effectiveFrom: string
 }
 
+export interface SignerData {
```

Comment:
We should move the rest of the app connector to keep interfaces on the `types` file. 👍